systems/WikibaseClient/05-Building_Block_View.md

```diff
@@ -12,6 +12,7 @@
 | [Linked Site Page Changes](#linked-site-page-changes)       | Inform a Repository of changes to pages that are linked to Repository Items      |
 | [Client Side Item Edits](#client-side-item-edits)           | Edit Item Data on the Repository                                                 |
 | [Special Pages](#special-pages)                             | WikibaseClient specific [Special Pages](../overview/12-Glossary.md#special-page) |
+| [Interwiki](#interwiki)                                     | Client side interactions with Interwiki links                                    |
```

Comment:
We say "WikibaseClient specific" above, but "Client side" here.
I believe these are meant to mean the same thing?